app/reflexes/enterprise_edit_reflex.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: false
+
+class EnterpriseEditReflex < ApplicationReflex
+  def remove_terms_and_conditions
+    @enterprise = Enterprise.find(element.dataset['enterprise-id'])
+    ability = can? :remove_terms_and_conditions, @enterprise
+    throw :forbidden unless ability
```

Comment:
It seems unnecessary to break over two lines.

```suggestion
    throw :forbidden unless can?(:remove_terms_and_conditions, @enterprise)
    
```
I also suggest including parentheses for the `can?` method call.  I _love_ that paretheses are optional in Ruby, but  find that it's clearer to read when the code is always formatted the same way (especially when there's more than one parameter like here).

And, I've added blank line because this functions like other "guard clauses" (`raise`, `return`, etc)